ci/tasks/parse-staging-release-tag.yml

```diff
@@ -0,0 +1,20 @@
+# This task parses the staging release candidate tag from an ecr resource and turns it into a *-staging release tag
+#
+# File written:
+#   parse-staging-release-tag/tag : The staging release tag (e.g. 123-release)
+#
```

Comment:
This comment isn't quite describing what's happening in the task, and we should probably update it to reflect the different reason for its use. 

The old task we copied this code from ([parse-perf-release-tag.yml](https://github.com/alphagov/pay-ci/blob/master/ci/tasks/parse-perf-release-tag.yml)) was designed to take a tag like `123-release` as an input, and output a different kind of tag: `123-perf`. Here, we're taking a tag `123-release`, and outputting the release number, `123`. Let's change the explanatory sentence above and the example of what's written to the output file.